Singer.API/Services/UserService.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Singer.Data;
+using Singer.DTOs;
+using Singer.Helpers;
+using Singer.Helpers.Exceptions;
+using Singer.Helpers.Extensions;
+using Singer.Models;
+using Singer.Services.Interfaces;
+
+namespace Singer.Services
+{
+   public class UserService : IUserService
+   {
+      private ApplicationDbContext _appContext;
+
+      public UserService(ApplicationDbContext appContext)
+      {
+         _appContext = appContext;
+      }
+
+      public async Task<T> CreateUserAsync<T>(T createUser) where T : CareUser
+      {
+         // TODO Use Usermanager
```

Comment:
Should this Todo be solved before merging?

Reply:
Same as above.